compute_funcx/endpoint/funcx_endpoint/version.py

```diff
@@ -1,6 +1,6 @@
 # single source of truth for package version,
 # see https://packaging.python.org/en/latest/single_source_version/
-__version__ = "2.5.0a0"
+__version__ = "2.5.0"
```

Comment:
I guess there's no harm to updating these, even if we don't push to PyPI

Reply:
Ah, yeah, just a find/replace "error."